### DISCUSSION TITLE
Support for prettier-plugin-sh

### DISCUSSION THREAD

**Discussion by AbdealiLoKo** [biomejs/biome#3784] [Rule suggestion]
I was trying to replace my use of prettier with biome, and realized that I have some plugins in prettier that biome does not support right now

Specifically I was looking for the shell script support. When I looked at the bash syntax check if did, it seemed quite simple and was wondering if biome can support this too

**Comment by dyc3** [comment]
Biome doesn't have support for bash. I don't think Biome will ever support bash natively. Plugin support is planned, but it is still in the very early stages. See: #2463
